### ISSUE TITLE
Vagrant support for VMware Fusion

### ISSUE THREAD

**Issue by mpepping** [gravitational/teleport#1536]
It would be nice to have VMware Fusion support in the Vagrant setup. Currently it's only possible to use Vagrant with either `libvirt` or Oracle Virtualbox, since the referenced `debian/contrib-jessie64` is only built for Virtualbox and libvirt.  

```
Bringing machine 'a-auth' up with 'vmware_fusion' provider...
Bringing machine 'b-auth' up with 'vmware_fusion' provider...
Bringing machine 'b-node' up with 'vmware_fusion' provider...
==> a-auth: Box 'debian/contrib-jessie64' could not be found. Attempting to find and install...
    a-auth: Box Provider: vmware_desktop, vmware_fusion, vmware_workstation
    a-auth: Box Version: >= 0
==> a-auth: Loading metadata for box 'debian/contrib-jessie64'
    a-auth: URL: https://vagrantcloud.com/debian/contrib-jessie64
The box you're attempting to add doesn't support the provider
you requested. Please find an alternate box or use an alternate
provider. Double-check your requested provider to verify you didn't
simply misspell it.

If you're adding a box from HashiCorp's Vagrant Cloud, make sure the box is
released.

Name: debian/contrib-jessie64
Address: https://vagrantcloud.com/debian/contrib-jessie64
Requested provider: ["vmware_desktop", "vmware_fusion", "vmware_workstation"]
```

**Comment by mpepping** [comment]
Extended the Vagrant setup with VMware Fusion support in #1537.

**Comment by klizhentas** [comment]
implemented by PR #1537
